src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -133,6 +133,7 @@ def __init__(
             safety_checker=safety_checker,
             feature_extractor=feature_extractor,
         )
+        self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
```

Comment:
@anton-l - think that's the safest actually

Reply:
Indeed, nice addition!